lib/__snapshots__/index.test.ts.snap

```diff
@@ -21730,6 +21730,122 @@ Object {
 }
 `;
 
+exports[`toGeoJSON schema.kml 1`] = `
```

Comment:
Looks like this PR is missing the `schema.kml` file that should be in the test fixtures directory.

Reply:
Oops, sorry! Pushed it now.